lib/improver/feels_like_temperature.py

```diff
@@ -0,0 +1,240 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2018 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Module containing feels like temperature calculation plugins"""
+
+import numpy as np
+
+from improver.psychrometric_calculations.psychrometric_calculations \
+     import WetBulbTemperature
+
+
+def calculate_wind_chill(temperature, wind_speed):
+    """
+    Calculates the wind chill from 10 m wind speed and temperature based on
+    the wind chill temperature index from the National Weather Service,
+    2001. While the wind chill index is often not used for wind speeds less
+    than 5 km/h, there is no upper limit for the wind speed.
+
+    Args:
+      temperature (iris.cube.Cube):
+        Cube of air temperatures (K)
+
+      wind_speed (iris.cube.Cube):
+        Cube of 10m wind speeds (m/s)
+
+    Returns:
+      wind_chill (iris.cube.Cube):
+        Cube of wind chill temperatures (K)
+
+    References:
+    Osczevski, R. and Bluestein, M. (2005). THE NEW WIND CHILL EQUIVALENT
+    TEMPERATURE CHART. Bulletin of the American Meteorological Society,
+    86(10), pp.1453-1458.
+
+    Osczevski, R. and Bluestein, M. (2008). Comments on Inconsistencies in
+    the New Windchill Chart at Low Wind Speeds. Journal of Applied
+    Meteorology and Climatology, 47(10), pp.2737-2738.
+    """
```

Comment:
I found the information in the papers very confusing (and I'm not the only one - the 2008 paper is actually attempting to clarify their 2005 paper). Therefore we need to add a small paragraph to explain the ideas of the papers. I suggest the below:

>Science background:
The 2005 Osczevski and Bluestein paper outlines their research and 
assumptions made, and their 2008 paper clarifies poorly explained 
sections of the first paper.

>A brief summary of their assumptions are given below:
Their model aims to determine a worst-case scenario of wind chill.
The wind speed "threshold" of 4.8kph (1.34m/s) stated in the 2005 papers
does not refer to a threshold placed on the input windspeed data, but instead
it is the walking speed of an average person. This is therfore used as the 
minimum wind speed used in their wind chill computer model, because even 
where wind speed is zero, a person would still experience wind chill by the
act of walking (their model of course assumes that the person is walking into
the wind). Their model also introduces a compensation factor where it assumes
that the wind speed at 1.5m (face level) is 2/3 that measured at 10m. It also
takes into account the thermal resistance of the human cheek skin.

>The equation outlined in their paper is also not the equation used in their
model (which was computationally expensive) but rather it is a linear
regression equation which mimics the output of their model where wind speeds
are greater than 3kph (0.8m/s) (clarified in their 2008 paper). Their
assumption being that lower wind speeds are usually not measured or reported
accurately anyway.

Reply:
I agree some explanation is necessary but I wonder whether there might be a better place to put this than the docstring.  @bayliffe @gavinevans can you comment?

Reply:
Added a description for wind chill and updated the description of apparent temperature to be slighlty clearer.